### ISSUE TITLE
.fit(shuffle=True)

### ISSUE THREAD

**Issue by patrick-ogrady** [fchollet/keras#4405]
if the number of epochs is one, does it shuffle before first run?

**Comment by varun-bankiti** [comment]
YES, you can see [here](https://github.com/fchollet/keras/blob/master/keras/engine/training.py#L826)
